display_information/add-graphics-with-renderer/src/main/java/com/esri/samples/add_graphics_with_renderer/AddGraphicsWithRendererSample.java

```diff
@@ -194,6 +211,28 @@ private Geometry makeHeartGeometry(Point centerOfHeart) {
     return heartShape.toGeometry();
   }
 
+  /**
+   * Create an ellipse shaded polygon.
+   *
+   * @return ellipse shaped polygon
+   */
+  private Polygon makeEllipse() {
+    // create parameters and set all the parameters
```

Comment:
create and set all parameters

Reply:
Might be nice to say something extra like:

 - The ellipse has a major axis of 400 kilometres, a minor axis of 200 kilometres and it is rotated at an angle of -45 degrees.

This is a nice explanation for someone who understands the maths behind ellipses.  Major / Minor makes better sense than 1 and 2.